docs/lesson-plans/week-1.md

```diff
@@ -0,0 +1,143 @@
+        - vim
+        - ls
+        - sudo
+        - man
+    - Build Tools
+        - NPM
+        - Maven
+        
+## Jumpstart Overview
+
+First, we'll be going over the general plan we have for Jumpstart over the next few months.
+
+### What is Jumpstart?
+
+From [our website](https://c4cneu.com/jumpstart).
+
+>Jumpstart is Code4Community's new program that focuses on teaching the basics of web 
+>development to beginners. 
+
+Over the next couple of weeks (we have 9-10 sessions planned including this one), we'll 
+be going through a lot of the fundamentals you'll need to know for developing an interactive 
+web application. Our aim is to help you understand and have the tools for developing a
+personal project of your own in the future.
+
+### Who is this program for?
+
+This program is honestly for anyone interested in software development or interested in the
+inner workings of a basic web application. Non-computer science majors are welcome! While a 
+background in computer programming is recommended, as long as you complete the recommended 
+preparation down below, you'll probably be well prepared for this course. We'll also be here 
+with dedicated tutors monitoring a special Slack channel and weekly office hours if you have
+any questions, no matter how silly they may seem.
+
+### How should I prepare?
+
+To prepare you for this course, we're recommending you take the [Java Codecademy 
+course](https://www.codecademy.com/learn/learn-java). The free course is more than enough to
+teach the understanding of Java we're expecting you to have. If you already know Java, then you'll
+probably be fine without taking it, but for anyone who feels they could benefit from the course,
+we highly recommend it. Also, it says it should take ~25 hours to complete, but that includes
+all of the paid content too, so expect it to take around 5-6 hours max.
+
+### What are we going to be building?
+
+Throughout this course, you'll have the opportunity (but you're not required) to build a project
+we've designed. It's a social media website similar to a blog, think of something like Reddit,
+with posts, comments, likes (we're calling them claps though), and a bit of other functionality.
+
+If you have an idea of a project you want to try and make instead of ours, feel free to join us
+and work on your project instead. Like it was said earlier, you're not required to make the 
+project. If you do end up working on your own project, be aware that the materials we give out 
+every week may not end up helping you as much as it would for our project, and you may encounter
+issues and concepts we won't be covering, however, feel free to use us as a resource for fixing
+and understanding them as they pop up.
+
+### What technologies will I be learning?
+
+During this time, we'll be teaching and providing demonstrations for a Java backend 
+using a Vert.x framework and Typescript frontend using a React framework with a PostgreSQL 
+database. The frontend
```

Comment:
Is this sentence suppose to continue? it just says The frontend

Reply:
Oops, I think you were looking at an old commit for the branch. At the top of the page under the Conversation tab (with the files changed tab selected), make sure the `Changes from` range says `all commits`

Reply:
Thanks for starting your review early!